changelog/fix-9996-currency-conversion-fee-phrasing

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Use "currency conversion fee", not "foreign exchange fee"
```

Comment:
Optional - make a brief mention of where these string changes would appear. e.g. "Use 'currency conversion fee', not 'foreign exchange fee' at various places including payment timeline, .... .

Reply:
Updated changelog to reflect this advice. :)